src/docs/sdk/performance/index.mdx

```diff
@@ -18,7 +18,11 @@ Reference implementations:
 
 This section describes the options SDKs should expose to configure tracing and performance monitoring.
 
-Tracing is enabled by setting either one of two new SDK config options, `tracesSampleRate` and `tracesSampler`. If not set, both default to `undefined`, making tracing opt-in.
+Tracing is enabled by setting any of three SDK config options, `enableTracing`, `tracesSampleRate` and `tracesSampler`. If not set, these options default to `undefined`, making tracing opt-in.
+
+### `enableTracing`
+
+This option shall replace the need to configure a `traceSampleRate` for many users. Sample rates shall be set at a default which is practical to the specific platform. Users may use the other options should their use case need it. The standard should be to set the default sample rate at 100%, and only working back if there are inherent concerns for that platform. Users should be able to send most if not all of their data and rely on Sentry server side processing of their data.
```

Comment:
Should the default of 100% (if `enableTracing=true`) (also) be mentioned further down under `tracesSampleRate`?

Reply:
I think the default should be "what makes sense". I would not try to commit to it setting a specific thing as otherwise people start depending on this. Ideally we eventually push down the right rates to the clients.

Reply:
This should align with how we explain `tracesSampleRate` or `tracesSampler`. Explain the options (true or false) and explain what it does (turns traces/performance on). Talking about sample rate should happen under `tracesSamplerRate` or `tracesSampler`

Reply:
If i understand you well I would agree, I will update this not to be worded as a replacement of another option but simply an option which turns of tracing & performace SDK functionalities with a default sample rate suitable  to the respective platform